static/docs/user-guide/external-data/amazon.md

```diff
@@ -0,0 +1,77 @@
+# External Data on Amazon S3
+
+Amazon S3 or Amazon Simple Storage Service is a service offered by Amazon Web
```

Comment:
let's remove this, no need to explain this

Reply:
better give an overview - we support both dependencies and outputs natively, mention important caveats if any etc ... applies to other storage types

Reply:
> let's remove this, no need to explain this

Agree